plugins/services/src/js/pages/__tests__/DeploymentsTab-test.js

```diff
@@ -43,7 +43,7 @@ describe('DeploymentsTab', function () {
         }
       ]
     });
-    DCOSStore.dataProcessed = true;
+    DCOSStore.serviceDataReceived = true;
```

Comment:
Since you have only defined a getter for `serviceDataReceived`, assigning a value to it should not be a legit operation. It currently works because we are mocking the `DCOSStore` component.

I am not going to block this PR since we need to get moving and this is not really prohibiting it from working properly, but please create an issue so we can tackle it at some point. 👍